NEO-TAT/tat_flutter
# Remove sensor plus package

## Description 
- Remove the `sensor_plus` package.

## How to Verify? 
- ENV: AOS API 31
1. create a course table home screen widget
2. let app become background, and go to that home screen widget and click it
3. app should back to foreground without crash

## Screenshots/GIF/Test Results (Optional) 
N/A
